glib/src/gobject/interface_info.rs

```diff
@@ -3,4 +3,24 @@
 #[derive(Debug)]
 #[doc(alias = "GInterfaceInfo")]
 #[repr(transparent)]
-pub struct InterfaceInfo(pub gobject_ffi::GInterfaceInfo);
+pub struct InterfaceInfo(pub(crate) gobject_ffi::GInterfaceInfo);
```

Comment:
```suggestion
pub struct InterfaceInfo(gobject_ffi::GInterfaceInfo);
```

as you would be from_glib_ptr_borrow_mut to construct them anyway, right?

Reply:
Seems fine to me like this, but a constructor `pub(crate)` would also work and reduce the code a bit